backend/tests/helpers/cleanups.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import elasticsearch
```

Comment:
new line between docs and imports

Reply:
This fixture is required to overwrite invenio es fixture which also deletes indexes.

Reply:
:+1: